[2022-10-22 01:17:18.827] [remoteagent] [info] Extension host agent started.
[2022-10-22 01:17:19.947] [remoteagent] [info] [127.0.0.1][d9e34ecb][ManagementConnection] New connection established.
[2022-10-22 01:17:20.225] [remoteagent] [info] [127.0.0.1][8a66369c][ExtensionHostConnection] New connection established.
[2022-10-22 01:17:20.230] [remoteagent] [info] [127.0.0.1][8a66369c][ExtensionHostConnection] <1550> Launched Extension Host Process.
[2022-10-22 01:17:30.034] [remoteagent] [info] [127.0.0.1][d9e34ecb][ManagementConnection] The client has disconnected gracefully, so the connection will be disposed.
[2022-10-22 01:17:30.045] [remoteagent] [info] [127.0.0.1][8a66369c][ExtensionHostConnection] <1550> Extension Host Process exited with code: 0, signal: null.
[2022-10-22 01:17:30.046] [remoteagent] [info] Last EH closed, waiting before shutting down
[2022-10-22 01:17:39.068] [remoteagent] [info] Got delay-shutdown request while in shutdown timeout, delaying
[2022-10-22 01:17:39.068] [remoteagent] [info] Cancelling previous shutdown timeout
[2022-10-22 01:17:40.143] [remoteagent] [info] [127.0.0.1][5682786d][ManagementConnection] New connection established.
[2022-10-22 01:17:40.636] [remoteagent] [info] [127.0.0.1][c332d0ab][ExtensionHostConnection] New connection established.
[2022-10-22 01:17:40.653] [remoteagent] [info] [127.0.0.1][c332d0ab][ExtensionHostConnection] <1704> Launched Extension Host Process.
[2022-10-22 01:17:58.542] [remoteagent] [error] [File Watcher (parcel)] Inotify limit reached (ENOSPC) (path: /home/ubuntu)
[2022-10-22 01:20:45.432] [remoteagent] [info] [127.0.0.1][5682786d][ManagementConnection] The client has disconnected gracefully, so the connection will be disposed.
[2022-10-22 01:20:45.528] [remoteagent] [info] [127.0.0.1][c332d0ab][ExtensionHostConnection] <1704> Extension Host Process exited with code: 0, signal: null.
[2022-10-22 01:20:45.528] [remoteagent] [info] Cancelling previous shutdown timeout
[2022-10-22 01:20:45.528] [remoteagent] [info] Last EH closed, waiting before shutting down
[2022-10-22 01:20:53.823] [remoteagent] [info] Got delay-shutdown request while in shutdown timeout, delaying
[2022-10-22 01:20:53.823] [remoteagent] [info] Cancelling previous shutdown timeout
[2022-10-22 01:20:54.656] [remoteagent] [info] [127.0.0.1][c4ccc5a4][ManagementConnection] New connection established.
[2022-10-22 01:20:55.151] [remoteagent] [info] [127.0.0.1][3c1b952e][ExtensionHostConnection] New connection established.
[2022-10-22 01:20:55.232] [remoteagent] [info] [127.0.0.1][3c1b952e][ExtensionHostConnection] <2120> Launched Extension Host Process.
[2022-10-22 01:21:08.246] [remoteagent] [error] [File Watcher (parcel)] Inotify limit reached (ENOSPC) (path: /home/ubuntu)
[2022-10-22 01:25:53.823] [remoteagent] [info] New EH opened, aborting shutdown
[2022-10-22 01:26:32.615] [remoteagent] [info] [127.0.0.1][c4ccc5a4][ManagementConnection] The client has disconnected gracefully, so the connection will be disposed.
[2022-10-22 01:26:32.657] [remoteagent] [info] [127.0.0.1][3c1b952e][ExtensionHostConnection] <2120> Extension Host Process exited with code: 0, signal: null.
[2022-10-22 01:26:32.657] [remoteagent] [info] Last EH closed, waiting before shutting down
[2022-10-22 01:26:42.077] [remoteagent] [info] Got delay-shutdown request while in shutdown timeout, delaying
[2022-10-22 01:26:42.078] [remoteagent] [info] Cancelling previous shutdown timeout
[2022-10-22 01:26:43.065] [remoteagent] [info] [127.0.0.1][544ca69a][ManagementConnection] New connection established.
[2022-10-22 01:26:43.545] [remoteagent] [info] [127.0.0.1][9c1ce001][ExtensionHostConnection] New connection established.
[2022-10-22 01:26:43.557] [remoteagent] [info] [127.0.0.1][9c1ce001][ExtensionHostConnection] <2569> Launched Extension Host Process.
[2022-10-22 01:26:46.593] [remoteagent] [error] [File Watcher (parcel)] Inotify limit reached (ENOSPC) (path: /home/ubuntu)
[2022-10-22 01:30:42.534] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-10-22 01:30:43.136] [remoteagent] [info] [127.0.0.1][9c1ce001][ExtensionHostConnection] The client has reconnected.
[2022-10-22 01:31:42.134] [remoteagent] [info] New EH opened, aborting shutdown
[2022-10-22 01:34:23.585] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-10-22 01:34:25.233] [remoteagent] [error] ["CodeExpectedError: Could not find pty on pty host","    at a._throwIfNoPty (/home/ubuntu/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:27:7252)","    at a.updateIcon (/home/ubuntu/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:27:3046)","    at Object.call (/home/ubuntu/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:16:8386)","    at c.onPromise (/home/ubuntu/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:15:5833)","    at c.onRawMessage (/home/ubuntu/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:15:5216)","    at /home/ubuntu/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:15:4502","    at m.invoke (/home/ubuntu/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at D.deliver (/home/ubuntu/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at O.fire (/home/ubuntu/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.X (/home/ubuntu/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:9:20837)","    at process.emit (node:events:526:28)","    at emit (node:internal/child_process:938:14)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-10-22 01:34:25.234] [remoteagent] [error] ["CodeExpectedError: Could not find pty on pty host","    at a._throwIfNoPty (/home/ubuntu/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:27:7252)","    at a.updateTitle (/home/ubuntu/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:27:2984)","    at Object.call (/home/ubuntu/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:16:8386)","    at c.onPromise (/home/ubuntu/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:15:5833)","    at c.onRawMessage (/home/ubuntu/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:15:5216)","    at /home/ubuntu/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:15:4502","    at m.invoke (/home/ubuntu/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at D.deliver (/home/ubuntu/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at O.fire (/home/ubuntu/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.X (/home/ubuntu/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:9:20837)","    at process.emit (node:events:526:28)","    at emit (node:internal/child_process:938:14)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-10-22 01:34:25.234] [remoteagent] [error] ["CodeExpectedError: Could not find pty on pty host","    at a._throwIfNoPty (/home/ubuntu/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:27:7252)","    at a.updateTitle (/home/ubuntu/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:27:2984)","    at Object.call (/home/ubuntu/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:16:8386)","    at c.onPromise (/home/ubuntu/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:15:5833)","    at c.onRawMessage (/home/ubuntu/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:15:5216)","    at /home/ubuntu/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:15:4502","    at m.invoke (/home/ubuntu/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at D.deliver (/home/ubuntu/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at O.fire (/home/ubuntu/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.X (/home/ubuntu/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:9:20837)","    at process.emit (node:events:526:28)","    at emit (node:internal/child_process:938:14)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-10-22 01:34:38.834] [remoteagent] [error] ["CodeExpectedError: Could not find pty on pty host","    at a._throwIfNoPty (/home/ubuntu/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:27:7252)","    at a.updateTitle (/home/ubuntu/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:27:2984)","    at Object.call (/home/ubuntu/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:16:8386)","    at c.onPromise (/home/ubuntu/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:15:5833)","    at c.onRawMessage (/home/ubuntu/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:15:5216)","    at /home/ubuntu/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:15:4502","    at m.invoke (/home/ubuntu/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at D.deliver (/home/ubuntu/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at O.fire (/home/ubuntu/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.X (/home/ubuntu/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:9:20837)","    at process.emit (node:events:526:28)","    at emit (node:internal/child_process:938:14)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-10-22 01:34:39.223] [remoteagent] [info] [127.0.0.1][544ca69a][ManagementConnection] The client has reconnected.
[2022-10-22 01:34:39.722] [remoteagent] [info] [127.0.0.1][9c1ce001][ExtensionHostConnection] The client has reconnected.
